From 6c1c9de145c253d0e67583a613b94f0b438f14df Mon Sep 17 00:00:00 2001
From: "Kevin F. Haggerty" <haggertk@lineageos.org>
Date: Mon, 19 Nov 2018 05:19:25 -0700
Subject: [PATCH 14/19] msm8974-common: sepolicy: Resolve
 hal_wifi_hostapd_default denials

* avc: denied { search } for name="wifi" dev="mmcblk0p25" ino=425157
  scontext=u:r:hal_wifi_hostapd_default:s0
  tcontext=u:object_r:wifi_vendor_data_file:s0 tclass=dir permissive=0

Change-Id: I56c1d43106911e24c8823c06a2637aecaeae5f6d
---
 sepolicy/common/hal_wifi_hostapd_default.te | 1 +
 1 file changed, 1 insertion(+)
 create mode 100644 sepolicy/common/hal_wifi_hostapd_default.te

diff --git a/sepolicy/common/hal_wifi_hostapd_default.te b/sepolicy/common/hal_wifi_hostapd_default.te
new file mode 100644
index 0000000..d3236b8
--- /dev/null
+++ b/sepolicy/common/hal_wifi_hostapd_default.te
@@ -0,0 +1 @@
+allow hal_wifi_hostapd_default wifi_vendor_data_file:dir search;
-- 
2.17.1

